pages.fr/common/ansible-playbook.md

```diff
@@ -0,0 +1,28 @@
+# ansible-playbook
+
+> Exécute les taches définies dans le playbook sur les machines distantes via SSH.
+> Plus d'informations : <https://docs.ansible.com/ansible/latest/cli/ansible-playbook.html>.
+
+- Exécute les taches d'un playbook :
+
+`ansible-playbook {{playbook}}`
+
+- Exécute les taches d'un playbook avec fichier d'inventaire spécifié :
+
+`ansible-playbook {{playbook}} -i {{fichier_d_inventaire}}`
+
+- Exécute les taches d'un playbook avec des variables supplémentaires définies via la ligne de commande :
+
+`ansible-playbook {{playbook}} -e "{{variable1}}={{valeur1}} {{variable2}}={{valeur2}}"`
+
+- Exécute les taches d'un playbook avec des variables supplémentaires définies dans un fichier JSON :
+
+`ansible-playbook {{playbook}} -e "@{{variables.json}}"`
+
+- Exécute les taches d'un playbook pour certain tags :
+
+`ansible-playbook {{playbook}} --tags {{tag1,tag2}}`
+
+- Exécute les taches d'un playbook en démarrant depuis une certaine tache :
```

Comment:
I just realized the difference between `tâche` for "task" and `tache` for  "dirty mark". 🤓 
See https://dictionnaire.lerobert.com/definition/tache

```suggestion
> Exécute les tâches définies dans le playbook sur les machines distantes via SSH.
> Plus d'informations : <https://docs.ansible.com/ansible/latest/cli/ansible-playbook.html>.

- Exécute les tâches d'un playbook :

`ansible-playbook {{playbook}}`

- Exécute les tâches d'un playbook avec fichier d'inventaire spécifié :

`ansible-playbook {{playbook}} -i {{fichier_d_inventaire}}`

- Exécute les tâches d'un playbook avec des variables supplémentaires définies via la ligne de commande :

`ansible-playbook {{playbook}} -e "{{variable1}}={{valeur1}} {{variable2}}={{valeur2}}"`

- Exécute les tâches d'un playbook avec des variables supplémentaires définies dans un fichier JSON :

`ansible-playbook {{playbook}} -e "@{{variables.json}}"`

- Exécute les tâches d'un playbook pour certain tags :

`ansible-playbook {{playbook}} --tags {{tag1,tag2}}`

- Exécute les tâches d'un playbook en démarrant depuis une certaine tache :
```

Reply:
Oh ! i always use `tache` in JIRA i should be more careful in the future 😅 
thank's for the review !